study-builder/studies/pancan/emails/consent-complete.en.html

```diff
@@ -0,0 +1,15 @@
+#set( $text = {
+    "subject": "Thank you for providing your consent",
+    "salutation_self": "Dear Participant",
+    "salutation_parent": "Dear Parent",
+    "thank_you": "Thank you for joining Count Me In and for giving us your consent for research.",
+    "what_to_expect": "We will next mail you a kit with instructions on how to provide a saliva sample.",
+    "what_to_expect_sample_request": "In some cases, we may also contact you about providing a blood sample (1 tube or 2 teaspoons per sample), which can be drawn at your physician's office, local clinic, or nearby lab facility. If applicable, we will provide detailed instructions on how to do this. We may obtain copies of {{~#if proxy_firstName~}}your child's{{~else~}}your{{~/if~}} medical records and, after reviewing them, may also request a small portion of stored tumor sample from the hospital where it is kept. Once we have analyzed {{~#if proxy_firstName~}}your child's{{~else~}}your{{~/if~}} requested samples, we will return any unused tumor samples to the pathology department that sent them to us.",
+    "next_steps": "If you have not yet provided information about where {{~#if proxy_firstName~}}your child has{{~else~}}you have{{~/if~}} received care for your cancer(s), you can complete the process here:",
+    "button_title": "Medical Release",
+    "contact": "For your convenience, we have attached a copy of your signed consent form. If you have questions at any point and would like to talk to a member of our study staff, please reach out to us at <a href='mailto:info@joincountmein.org' target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>info@joincountmein.org</a> or call <a href='tel:000-000-0000' style='color: #7154ff; text-decoration: none; cursor: pointer;>000-000-0000</a>.",
```

Comment:
It make sense to put contants and social media links externaly 
(see [example](https://github.com/broadinstitute/ddp-study-server/pull/864/files#:~:text=%22social%22%3A%20%7B,instagram.com/countmein%22) )

Reply:
I'll get this updated once PR #864 containing the substitutions is merged in.

Reply:
Incorporated use of the shared contact & social media details. Fixed in d48ce8f2e8926b40d6574c139feb5da5450ef712